src/components/App.tsx

```diff
@@ -1,15 +1,34 @@
 import React from "react";
 import { LoginForm } from "./LoginForm";
+import { Dropdown } from "./Dropdown";
+import { DropdownCourse } from "./DropdownCourse";
 
 export const App: React.FC = () => {
   const [isLoggedIn, setIsLoggedIn] = React.useState(false);
-
+  const course = ["one", "two", "three"];
   return (
     <div className="w-96 h-96 bg-white p-4">
       {!isLoggedIn ? (
         <LoginForm onLogin={() => setIsLoggedIn(true)} />
       ) : (
+        <>
+      <div className="mb-1 text-[12px] font-normal text-[#959494] leading-[110%]">
```

Comment:
Is `#959494` a theme color? If so, it should be defined in our tailwind file and brought in here under a `primary`, `secondary`, etc class name, so we won't be repeating ourselves throughout the entire application.